Assets/BossRoom/Scripts/Server/Game/Character/ServerCharacter.cs

```diff
@@ -189,6 +189,11 @@ public void ReceiveHP(ServerCharacter inflicter, int HP)
             }
             else
             {
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+                // Don't apply damage if god mode is on
+                if (NetState.NetworkLifeState.IsGodMode.Value) return;
```

Comment:
Just a stylistic, but consistent across the codebase, suggestion to wrap the return in brackets in new lines.